BOUNTY.md

```diff
@@ -1,24 +1,33 @@
+[![license](https://img.shields.io/github/license/musicoin/desktop.svg?style=for-the-badge)](https://github.com/Musicoin/desktop/blob/master/LICENSE)
```

Comment:
I don't think the badges are necessary on the bounty page, we can remove them.

Reply:
They're in the updated readme, so that makes sense.